### DISCUSSION TITLE
Resource Manager v9.11.0

### DISCUSSION THREAD

**Discussion by DNNMonster** [dnnsoftware/Dnn.Platform#5436] [Q&A]
Does the new Resource Manager v9.11.0 work for registered users in user mode?

I'm getting http 401 errors on requests to the ResourceManager GetAddFolderTypeUrl api endpoint for non admins.  I think I see the DNNAuthorize attribute causing it but I am wondering if others are also experiencing this.

No files or folders render for registered users.

**Comment by valadas** [comment]
I have not tested a lot in user mode, it is supposed to work but not sure if anyone tested that portion enough. Are you saying it just happens or are non-admins actually trying to manage folder types (they should no be able to do that by design)...

If it just happens without them trying to do anything special, I would log an issue about it.

**Reply by mitchelsellers** [reply]
@valadas I'm going to reach out to you about this directly

**Comment by DNNMonster** [comment]
I may have some environment specific issues going on and there may be some changes that are coming in 9.11.1 that help out.  I tried a couple of things to fix it but they were not successful.  Pretty sure it is best to wait for the next release.  I like the UI changes so it'll be cool to get it going.
